organisaatio-ui/src/components/HttpRequests.tsx

```diff
@@ -0,0 +1,8 @@
+import { Ryhma } from '../types/types';
```

Comment:
Would it make sense to 'hide' axios stuff within this module? E.g. instead of returning `AxiosResponse` one could only return `Ryhma` -> Other parts of the system would then only operate on 'domain' objects instead of the need to know anything about the data interchange?

Reply:
Good idea. I will do this improvement when i start making adding new ryhma functionality